readme.md

```diff
@@ -57,45 +90,61 @@ testing
 }
 ```
 
-### Size Benefits
+#### This output represents a npm packages ready to be used without locally installing them!
+
+---
+
+## Comparisions
 
-One very significant upside of using deps using the `loader` and `import-map`. The dependencies from `test.js` loads
+Additionally, `@jspm/node-importmap-loader` can produce significant size benefits
+
+#### With regular npm install
 
 ```sh
-du -sh -I @jspm -I fs-extra -I node-fetch node_modules
+du -hs node_modules
```

Comment:
`-sh` is for mac linux machines. Maybe both can co-exist. And next any deps the loader uses need to be omitted. Or else we are not calculating thee true size of the application deps.